.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
```

Comment:
Not a blocker, but there's an opportunity to update [1] so it's in the docs.

[1] https://github.com/pokt-network/poktroll/tree/main/docusaurus/docs/develop